src/draw/nxp/pxp/lv_pxp_osa.c

```diff
@@ -66,11 +66,8 @@ static void _pxp_wait(void);
  *  STATIC VARIABLES
  **********************/
 
-#if defined(SDK_OS_FREE_RTOS)
-    static SemaphoreHandle_t xPXPIdleSemaphore;
-#endif
-#if defined(__ZEPHYR__)
-    static K_SEM_DEFINE(s_pxpIdleSem, 0, 1);
+#if LV_USE_OS
```

Comment:
PXP can work with or without FreeRTOS support.
PXP in baremetal mode means that SDK_OS_FREE_RTOS = 0.

While LV_USE_OS is a feature add in LVGL for drawing parallelism. Those two defines have a different meaning.
I will give you an example:
```
SDK_OS_FREE_RTOS = 1
LV_USE_OS = LV_OS_NONE
``` 
This means that we will use LVGL single thread mode (same as LVGL v8) but PXP will have FreeRTOS support.

Reply:
Understood, so this destinction needs to be handled.

Reply:
We can fix later this behavior. If you fix-up the inclusions then you can merge this PR as is.